experiments/AMIP/components/atmosphere/climaatmos_init.jl

```diff
@@ -100,11 +100,7 @@ function get_field(sim::ClimaAtmosSimulation, ::Val{:liquid_precipitation})
     sim.integrator.p.precipitation.col_integrated_rain .* ρ_liq # kg/m^2/s
```

Comment:
Ahh, this is probably it:

```suggestion
    sim.integrator.p.precipitation.col_integrated_rain # kg/m^2/s
```